### PR TITLE
Show the correct information on the StoreDetails page.

### PR DESCRIPTION
**Pull request by officialcwang**
Wrote this code to work with hard-coded data, since the APIs are being changed. Resolved some of the warnings.
details

I'm also getting the following warning: "Warning: Can't call setState on a component that is not yet mounted. This is a no-op, but it might indicate a bug in your application. Instead, assign to `this.state` directly or define a `state = {};` class property with the desired state in the Wrapper component." Not sure what's wrong?

### CODE REVIEW

File: capstone/client/src/components/StoresPage.js [hunk 1]
```diff
@@ -14,64 +14,59 @@
...
+      stores : []
     }
+    this.getStores = this.getStores.bind(this);
+  }
 
-    componentWillMount = () => {
-        // Get Stores from database.
-        this.getStores();
-    }
+  componentWillMount = () => {
```
Comment (chen-dawn) [inline capstone/client/src/components/StoresPage.js:41]: Change this to componentDidMount

File: capstone/client/src/components/StoresPage.js [hunk 2]
```diff
@@ -14,64 +14,59 @@
...
-      this.state = {
-          stores : []
-      }
-      this.getStores = this.getStores.bind(this);
+  constructor(props) {
+    super(props)
+    this.state = {
+      stores : []
     }
+    this.getStores = this.getStores.bind(this);
```
Comment (chen-dawn) [inline capstone/client/src/components/StoresPage.js:38]: Move this line to componentDidMount, that should fix the error

File: capstone/client/src/components/StoreDetailCards.js [hunk 3]
```diff
@@ -15,43 +15,47 @@
...
-        )
-    }
+  render() {
+    const storeDetailCards = this.props.stores.map((store) => (
+      <div>
+        <Typography variant='h4'>{store.storeName}</Typography>
+        <Typography variant='h6'>Address: {store.address}</Typography>
+        <Grid container alignItems="stretch">
+          <Grid item component={Card} xs>
+            <Map id="google-map" google={this.props.google} zoom={14} style={{width: '25%', height: '25%'}}>
```
Comment (chen-dawn) [inline capstone/client/src/components/StoreDetailCards.js:32]: Move the Style to CSS file

Comment (officialcwang) [reply capstone/client/src/components/StoreDetailCards.js:32]: The styles.css file is really weird. In pages that don't import it (like ListPage.js), the style applies, but regardless of whether I import it or not in this file, Google Maps style doesn't change unless I put the style changes in the file (StoreDetailCards.js) itself. For now, I put the json style in a separate variable in the render function, until we can look at it later?

File: capstone/client/src/components/StoresPage.js (hunk 2 above)
Comment (officialcwang) [reply capstone/client/src/components/StoresPage.js:38]: Hm, the error is still there.

**Pull request merged**